### ISSUE TITLE
Enhancement: `Add to playlist` should be opened after creating a playlist

### ISSUE THREAD

**Issue by Abestanis** [nt4f04uNd/sweyer#45]
<details>
<summary>After creating a playlist, it currently takes two additional clicks to add tracks to the new playlist.</summary>

![Creating a playlist](https://user-images.githubusercontent.com/6966049/169151332-affebab8-d95f-4898-903f-4dee8e64c2ad.gif)
</details>

Given that most people would probably like to fill a newly created playlist with songs, the `Add to playlist` screen should automatically open for the new playlist after the user clicks `Create` in the naming dialog.

**Comment by nt4f04uNd** [comment]
I agree this behavior can be improved, but just opening this screen without asking would be a bit annoything, I'd rather show a toast that would suggest opening this screen to, like we have in `_SaveQueueAsPlaylistAction`. This is compromise that would not annoy and improve UX at the same time.

**Comment by Abestanis** [comment]
I don't feel like it would be annoying at all, adding items to the newly created playlist is just the next logical step. Can you think of an example where the user wouldn't want to add something to the new playlist?

**Comment by nt4f04uNd** [comment]
> Can you think of an example where the user wouldn't want to add something to the new playlist?

User creates a playlists and fills it in later, in a way that they prefer to do that. There is lots of ways to do this - from any screen in the app, in fact.

> I don't feel like it would be annoying at all, adding items to the newly created playlist is just the next logical step.

From UX point of view it seems like a screaming "you should add tracks NOW", while mildly suggesting a user to fast-jump to do this seems like a behavior that would suit all.
